### ISSUE TITLE
Error message installing plugin from zip file: "Fail to load plugin descriptor file on intellij-elixir-18.0.1.zip"

### ISSUE THREAD

**Issue by aselder** [KronicDeth/intellij-elixir#3623]
**Describe the bug**
Error message installing plugin from zip file

**To Reproduce**
1. Go to the GitHub release page 
2. Download the latest zip file
4. Go to IntelliJ 2024.1.4, attempt install plugin from disk
5. Get error: "Fail to load plugin descriptor file on intellij-elixir-18.0.1.zip"

**Expected behavior**
Plugin loaded

**Screenshots**
<img width="417" alt="Screenshot 2024-08-04 at 1 22 32 PM" src="https://github.com/user-attachments/assets/8094b3be-ef57-40e2-b65b-0c3595f3f48d">

**Desktop:**
 - OS: macOS
 - Version 14.6

**Erlang:**
- Installer N/A
- Version N/A

**Elixir:**
- Installer N/A
- Version N/A

**Plugin:**
- Version 18.0.1

**Additional context**
Plugin isn't showing in the Plugin search list, which is why I tried to install from disk.

**Comment by patcow** [comment]
Also having this issue on 14.5

**Comment by joshuataylor** [comment]
I've fixed the release, I also added a note around current issues.

We're going to submit a request for an organisation, we'll try and get this sorted in the next few days.

This way multiple people can manage releases.

We can also get a few things setup with this:

1. [Exception analyzer](https://plugins.jetbrains.com/docs/marketplace/exception-analyzer.html#what-metadata-is-sent) - users can submit errors and we can see them, opt in. This is the same process as submitting it to the GH issue tracker.
2. [Custom release channels](https://plugins.jetbrains.com/docs/marketplace/custom-release-channels.html#uploading-a-plugin-update-to-the-custom-channel), allowing users to automatically update to test releases. I would like BETA & Nightly builds here.
3. [Add documentation and other info the main plugin page](https://plugins.jetbrains.com/docs/marketplace/custom-pages.html)

Once the stable version has been submitted, it has to go through the JetBrains plugin review process.
